README.md

```diff
@@ -1,2 +1,16 @@
 # on-skupack
 Maintain SKU specific package content such as templates and other static files.
+
+# Quick Start
+* Build SKU package
+
+```
+$ ./build-package.bash <sku_pack_directory> <subname>
+```
+
+* Register a new SKU with the package
+```
+$ curl -X POST --data-binary @sku_pack_directory_subname.tar.gz http://localhost:8080/api/common/skus/pack
```

Comment:
Rebase and run master which includes this: https://github.com/RackHD/on-http/commit/e7c3b712f6895e903652a73cfefd8e2cef857db4.  Both calls will work but it was pointed out that multipart/form-data is more correct; especially for large files.

Good catch on the documentation.  I'll update readthedocs.